openehr-rm/src/main/java/com/nedap/archie/rm/changecontrol/VersionedObject.java

```diff
@@ -42,11 +43,12 @@ public void setOwnerId(ObjectRef ownerId) {
         this.ownerId = ownerId;
     }
 
-    public DvDateTime getTimeCreations() {
+    public DvDateTime getTimeCreated() {
         return timeCreated;
     }
 
-    public void setTimeCreations(DvDateTime timeCreated) {
+    @JsonAlias({"time_created"})
```

Comment:
Should this be the old name instead of the new name?

Reply:
oops, i'll change that, thanks